frontend/packages/console-shared/src/components/form-utils/FormContent.scss

```diff
@@ -0,0 +1,6 @@
+.ocs-form-content {
+  margin-bottom: 5em;
+  &__error {
+    margin-bottom: 10em;
```

Comment:
Can we use patternfly spacer variables for margin?

Reply:
Updated.